pipeline/inputs/process-exporter-metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+description: >-
+  A plugin based on Process Exporter to collect process level of metrics of system
+  metrics
+---
+
+# Process Exporter Metrics
+
+[Prometheus Node Exporter](https://github.com/prometheus/node_exporter) is a popular way to collect system level metrics from operating systems, such as CPU / Disk / Network / Process statistics. Fluent Bit 2.2 includes process exporter metrics plugin that builds off the Prometheus design to collect process level metrics without having to manage two separate processes or agents.
+
+The initial release of Process Exporter Metrics refers a process group level collector to collect metrics available from [the 3rd party implementation of Prometheus Process Exporter](https://github.com/ncabatoff/process-exporter) and we plan to expand them over time if needed.
+
+**Important note:** Metrics collected with Process Exporter Metrics flow through a separate pipeline from logs and current filters do not operate on top of metrics.
+
+This plugin is only supported on Linux based operating systems.
```

Comment:
What about macOS?

Reply:
macOS does not have procfs. So, we can support it only for Linux-based operating systems.

Reply:
Can we add a comment to that effect? I would consider macOS to be in the "Linux-based" terminology so it is good to be clear it is not supported.

Reply:
macOS does not based on Linux-based technology. It *is* based on BSD 4.2 and Mach technology. filesystem is built on top of IOKit which is made by Apple actually....

Reply:
Sure, I understand but I think it's easy to mistake it for "Linux-based" so be good to ensure we clarify this is not supported on macOS. That's my only point, clarify it is not supported because of the lack of `procfs`.

Reply:
Sounds reasonable. I added descriptions for procfs and won't supported it for macOS: https://github.com/fluent/fluent-bit-docs/pull/1198/commits/70ad3c2c905d1ce24578873ca5f09eb054e94266